pkg/asset/installconfig/openstack/validate.go

```diff
@@ -9,6 +11,10 @@ import (
 
 // Validate validates the given installconfig for OpenStack platform
 func Validate(ic *types.InstallConfig) error {
+	if skip := os.Getenv("OS_SKIP_PREFLIGHT_VALIDATIONS"); skip == "1" {
```

Comment:
while we expect the user to know what they're doing™, I think a logrus.Warnf would maybe make UX a bit better.